aspen/website.py

```diff
@@ -68,15 +69,19 @@ def redirect(self, location, code=None, permanent=False, base_url=None, response
         permanent is True and 302 (Found) if it is False. If url doesn't start
         with base_url (defaulting to self.base_url), then we prefix it with
         base_url before redirecting. This is a protection against open
-        redirects. If you provide your own response we will set .code and
-        .headers['Location'] on it.
+        redirects. If base_url is not '' and location is not an absolute path,
```

Comment:
I think we want to allow relative path redirects, in which case, with a non-empty `website.base_url`, the `base_url` parameter becomes the way to explicitly whitelist a relative path or absolute URL redirect. See https://github.com/gratipay/gratipay.com/commit/ebce89f10244a65d52e8eb0c07a015d8eea7019a for relative path example usage, and https://github.com/gratipay/gratipay.com/commit/fca0cf1e4d990db6df7c80901541e6022d53013e for absolute URL.